parsimonious/expressions.py

```diff
@@ -262,6 +263,7 @@ class TokenMatcher(Literal):
     This is for use only with TokenGrammars.
 
     """
+
```

Comment:
I was going to just add on a commit to fix it myself, but I finally have to admit to myself that I'm never going to get around to it: why add the extra blank lines here? Elsewhere in Parsimonious I hew to the old recommendations of PEP 257, which were a blank line before the `"""` and none afterward. (They've since edited it.) Let's stick to that, just for consistency, unless there's good reason otherwise.